proto/xudrpc.proto

```diff
@@ -66,7 +66,38 @@ service XUDService {
 
 message GetInfoRequest {}
 message GetInfoResponse {
-  lnrpc.GetInfoResponse lnd = 1 [json_name = "lnd"];
+    int32 peers = 1 [json_name = "peers"];
+    int32 pairs = 2 [json_name = "pairs"];
```

Comment:
Let's go with `num_peers` and `num_pairs` or similar to clarify that these are counts.